setup.py

```diff
@@ -7,6 +7,8 @@
 libdir = os.path.abspath(os.path.join(scipoptdir, 'lib'))
 libname = 'scip'
 
+print(scipoptdir)
+
```

Comment:
Why is this necessary?

Reply:
Not necessary, but I thought useful for the user to know what the environment variable is set to. I added this when I was having trouble with linking the scipoptdir.

If you don't like it, we can remove it.

Reply:
I would rather not have it. The `setup.py` should be as minimal/clean as possible.

Reply:
I have updated this now